.storybook/main.ts

```diff
@@ -25,16 +26,47 @@ module.exports = {
 
     const customStorybookConfig = defineConfig({
       base,
+      define: {
+        'process.browser': 'true',
+      },
       plugins: [
+        // TODO: The following will fix SynapseFormWrapper in development mode but will break the storybook static build
+        // Need to figure out why the polyfill breaks the build
+        // nodePolyfills({
+        //   include: 'node_modules/@apidevtools/json-schema-ref-parser/**/*.js',
+        // }),
         svgr(),
         viteExternalsPlugin({
           react: 'React',
           'react-dom': 'ReactDOM',
           'sql-parser': 'SQLParser',
-          '@sage-bionetworks/rjsf-core': 'JSONSchemaForm',
-          '@rjsf/core': 'JSONSchemaForm',
         }),
       ],
+      resolve: {
+        alias: {
+          stream: 'stream-browserify',
+          buffer: 'buffer/',
+          timers: 'timers-browserify',
+        },
+      },
+      optimizeDeps: {
+        include: [
+          '@storybook/react/dist/esm/client/docs/config',
+          '@storybook/react/dist/esm/client/preview/config',
+          '@storybook/addon-links/preview.js',
+          '@storybook/addon-docs/preview.js',
+          '@storybook/addon-actions/preview.js',
+          '@welldone-software/why-did-you-render',
+          '@storybook/addon-backgrounds/preview.js',
+          '@storybook/addon-measure/preview.js',
+          '@storybook/addon-outline/preview.js',
+          '@storybook/addon-interactions/preview.js',
+          'buffer/',
+        ],
+      },
```

Comment:
This should make storybook faster to start up (i.e. will use cached artifacts more often) and more reliable